Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.17 as builder
+
+WORKDIR /go/src/temporalite
+ADD . .
+RUN go mod download
+RUN go get -d -v ./...
+RUN go build -o /go/bin/temporalite ./cmd/temporalite
+
+FROM gcr.io/distroless/base-debian11
+
+COPY --from=builder /go/bin/temporalite /
+EXPOSE 7233
+ENTRYPOINT ["/temporalite", "start", "-f", "/tmp/temporalite.db"]
```

Comment:
Should we also bind to 0.0.0.0 for convenience?

Reply:
Or borrow from the upstream entrypoint script: https://github.com/temporalio/temporal/blob/master/docker/entrypoint.sh

Reply:
These are great ideas, I will see where I can find the time to sequence in the work.

Reply:
> Or borrow from the upstream entrypoint script: https://github.com/temporalio/temporal/blob/master/docker/entrypoint.sh

I am not sure that is possible using distro-less given there is no shell.

Alpine has its own issues given the differences between musl and glibc.

I am using a combo container for my testing is based on Debian bullseye and has a copy of tcl.
The ENTRYPOINT is easily overridden from docker cli or even docker-compose.

Reply:
This is updated now, a simple sane default.

Updated the PR comment to show execution commands.